backend/src/main/java/matches/organizer/service/MatchService.java

```diff
@@ -43,6 +44,22 @@ public List<Match> getMatches() {
         return matchRepository.getAll();
     }
 
+    public List<Player> registerNewPlayer(UUID id, User user, String phone, String email) {
+        var newMatch = matchRepository.get(id);
+
+        if(newMatch != null) {
+            addPlayerToMatch(newMatch, user, phone, email);
+            matchRepository.update(newMatch);
+            System.out.println("Se agrega un player al match" + id
+            + "\ncon el alias: " + user.getAlias()
+            + "\ncon el telefono: " + phone
+            + "\ncon el email " + email);
+            return newMatch.getPlayers();
+        } else {
+            return new ArrayList<>();
```

Comment:
Si no se encuentra el match, debe retornar un 404 Not found

Reply:
Agregado.